src/routes/example.js

```diff
@@ -0,0 +1,26 @@
+import Boom from '@hapi/boom'
+import { findAllExampleData, findExampleData } from '../example-find.js'
+
+const example = [{
```

Comment:
Exporting as collection of routes rather than plugins.   Aligns better to the pattern within Hapi docs and the majority of repos across Defra.

Reply:
Worth noting the the use of plugins is to future proof for more complex routes. fEx - https://github.com/DEFRA/cdp-portal-frontend/blob/main/src/server/services/about/routes.js

I wonder if theres a happy medium we can come up with here?

Also worth noting the import is not using the `babel-plugin-module-resolver` import of `~/src/`

Reply:
I think this highlights the main challenge in creating any template.  Getting the balance right between what teams definitely need and what teams might need.

If we make the templates too minimal then teams frequently ask for support and guidance for how to solve a common problem.  If we try to cover more potential scenarios, then they become difficult to understand and teams frequently just delete stuff to simplify it again.

If we don't have any complex routes in the template, then I'd suggest not having a more complex setup to support them.

If teams need that complexity, then they can add it when that complexity is needed rather than the default being complex.